public/css/main.css

```diff
@@ -0,0 +1,934 @@
+  background-color: rgb(124,131,135);
+}
+article h1:before, .services h3:before, .about h2:before, .team h1:before, .clients h2:before {
+  margin-left: -50%;
+  text-align: right;
+}
+/*&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+    Bottom-notch styles
+
+&&&&&&&&&&&&&&&&&&&&&&&&&&&*/
+.header-notch:after, .header-notch:before {
+  position: absolute;
+  content: "";
+  width: calc(50% - 88px);
+  height: 0;
+  border-bottom: 20px solid #fd6632;
+  border-left: 85px solid transparent;
+  left: calc(50% + 3px);
+  bottom: 0;
+}
+.header-notch:before {
+  border-left: none;
+  border-right: 85px solid transparent;
+  left: 0;
+}
+.orange-notch, .white-notch, .black-notch {
+  position: relative;
+  left: 50%;
+  bottom: 16px;
+}
+.orange-notch:before,
+.orange-notch:after, .white-notch:before,
+.white-notch:after, .black-notch:before,
+.black-notch:after {
+    position: absolute;
+    content: "";
+    left: calc(50% + 10px);
+    bottom: -41px;
+    width: 40px;
+    height: 100px;
+    background-color: #fd6632;
+    -moz-border-radius: 0 180px 50px 50px;
+    border-radius: 0 100px 50px 50px;
+    -webkit-transform: rotate(-75deg);
+       -moz-transform: rotate(-75deg);
+        -ms-transform: rotate(-75deg);
+         -o-transform: rotate(-75deg);
+            transform: rotate(-75deg);
+    -webkit-transform-origin: 0 100%;
+       -moz-transform-origin: 0 100%;
+        -ms-transform-origin: 0 100%;
+         -o-transform-origin: 0 100%;
+            transform-origin: 0 100%;
+}
+.orange-notch:after, .white-notch:after, .black-notch:after {
+    left: calc(50% - 55px);
+    -moz-border-radius: 100px 0 50px 50px;
+    border-radius: 100px 0 50px 50px;
+    -webkit-transform: rotate(75deg);
+       -moz-transform: rotate(75deg);
+        -ms-transform: rotate(75deg);
+         -o-transform: rotate(75deg);
+            transform: rotate(75deg);
+    -webkit-transform-origin: 100% 100%;
+       -moz-transform-origin: 100% 100%;
+        -ms-transform-origin: 100% 100%;
+         -o-transform-origin: 100% 100%;
+            transform-origin: 100% 100%;
+}
+.white-notch:before, .white-notch:after {
+  background-color: #fff;
+  z-index: 1;
+}
+.black-notch:before, .black-notch:after {
+  background-color: #2a2a2e;
+}
+@media screen and (min-width: 50em) {
+  .orange-notch:before,
+  .orange-notch:after, .white-notch:before,
+  .white-notch:after, .black-notch:before,
+  .black-notch:after {
+    bottom: -38px;
+  }
+}
+/*&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+    Top navigation bar
+
+&&&&&&&&&&&&&&&&&&&&&&&&&&&*/
+.top-nav {
+  padding: auto 0;
+}
+.top-nav li {
+  text-align: center;
+  border: 1px solid rgba(0,0,0,0.2);
+  padding: .5em;
+  font-weight: 300;
+}
+/* flex nav wrapper - adapted from flexboxin5.com */
+.flex-navbar {
+  display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
+  display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
```

Comment:
Old browsers are outside the scope of this assignment. It's fine with me that you included these, but please be sure you understand what properties are needed for new vs. legacy flex-box; no need to include legacy declarations if you don't need to support legacy browsers.